test/ui/helpers/container/InsightUI.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
```

Comment:
`test` folder should include only tests. Libraries should be in `lib` or `src`. Let's move `test/ui/helpers` to `lib/test`.

Reply:
updated structure

Reply:
angular has the same structure as mine initial
https://github.com/angular/angular.js/blob/master/test/e2e/tools/util.js